sdk/storage/storage-blob-changefeed/src/BlobChangeFeedClient.ts

```diff
@@ -0,0 +1,119 @@
+import { BlobServiceClient } from "@azure/storage-blob";
+import { PagedAsyncIterableIterator, PageSettings } from "@azure/core-paging";
+import { BlobChangeFeedEvent } from "./models/BlobChangeFeedEvent";
+import { ChangeFeedFactory } from "./ChangeFeedFactory";
+import { ChangeFeed } from "./ChangeFeed";
+import { CHANGE_FEED_DEFAULT_PAGE_SIZE } from "./utils/constants";
+
+export interface ChangeFeedGetChangesOptions {
```

Comment:
This should extend a common option bag with a tracingOptions property. and we can do tracing in the private method.

Reply:
ideally we would like to create a span when sending request to the service.

Reply:
I think we just need to pass the tracing options to the underlying blob clients' methods?

Reply:
@xirzec do you have any recommendation on doing tracing when we call blob client methods under the hood?

Reply:
Yes, generally every time we make a request to the service we want to create a span in the client that functions as the parent of the span made in the http request.

Are they any other standard properties from OperationOptions that would apply here, like timeout?

Reply:
> Are they any other standard properties from OperationOptions that would apply here, like timeout?

In storage packages, we don't expose timeout in convenience layer APIs' options. For example:
```
export interface BlobDownloadOptions extends CommonOptions {
  /**
   * An implementation of the `AbortSignalLike` interface to signal the request to cancel the operation.
   * For example, use the &commat;azure/abort-controller to create an `AbortSignal`.
   *
   * @type {AbortSignalLike}
   * @memberof BlobDownloadOptions
   */
  abortSignal?: AbortSignalLike;
...
```

```
export interface CommonOptions {
  /**
   * Options to configure spans created when tracing is enabled.
   */
  tracingOptions?: OperationTracingOptions;
}
```

I agree we need to add support for `abortSignal` and `tracingOptions`. We plan to do that in the next GA release for Dec19. Is this ok with you?

Reply:
Sounds good.  Please log an issue to track the work.